pkg/s3wrapper/client.go

```diff
@@ -55,6 +55,12 @@ type Config struct {
 
 // NewS3Client creates new s3 client using default config along with defined env variables
 func NewS3Client(cfg *Config, logger *logrus.Logger) *S3Client {
+	new_url, err := FixEndpointURL(cfg.S3EndpointURL)
```

Comment:
Why do you need it here? there are no issues with go client no?

Reply:
Maybe? I just realized that I added this as a precaution before we got to test this part of the code.
I can remove it if you think the go client will accept at hostname as the endpoint url.

Reply:
lets try without it. no reason to add code if it's not needed